docs/migration-guide.md

```diff
@@ -2,11 +2,196 @@
+middleware in your `.neutrinorc.js` with:
+
+```js
+module.exports = {
+  options: {
+    debug: true
+  }
+};
+```
+
+- **BREAKING CHANGE** `@neutrinojs/html-loader` used by the web-related presets now parse URLs defined in link tags
+in the HTML template [#943](https://github.com/neutrinojs/neutrino/pull/943).
+- **BREAKING CHANGE** The `@neutrinojs/hot` [#981](https://github.com/neutrinojs/neutrino/pull/981) and `@neutrinojs/env`
+[#983](https://github.com/neutrinojs/neutrino/pull/983) middleware have been removed and their relevant
+functionality is now configured directly by its previously-consuming presets.
+- **BREAKING CHANGE** `@neutrinojs/node` now compiles to the supported version of Node.js being used to run the build
+[#991](https://github.com/neutrinojs/neutrino/pull/991). Previously this was always defaulted to a particular Node.js v6
+version. For example, if you use Node.js v8 to run webpack, Neutrino will tell Babel to compile your project to support
+back to Node.js v8.
+- **BREAKING CHANGE** Various build outputs and settings used by Neutrino were driven by the `NODE_ENV` environment
+variable [#972](https://github.com/neutrinojs/neutrino/pull/972). How the environment variable drives some of these
+options has changed and may cause output that is different than in v8. For example, sometimes output was controlled
+when the value *was set to `development`*, but now may be done when the value *is not `production`*.
+- **BREAKING CHANGE** The merging of Babel options has been updated to *not merge array options*, but instead
+overwrite them [#1022](https://github.com/neutrinojs/neutrino/pull/1022).
+- **BREAKING CHANGE** `@neutrinojs/eslint` and its dependent middleware now require ESLint v5, which may mean breaking
+changes to rule configuration and any `parserOptions` modifications [#1025](https://github.com/neutrinojs/neutrino/pull/1025).
+- **BREAKING CHANGE** The `options.mains` Neutrino option can now also accept an object per entry point for setting
+individual options for html-webpack-plugin [#1029](https://github.com/neutrinojs/neutrino/pull/1029). The previous API
+for setting these properties as strings is still supported. This also introduces a breaking API change as accessing an
+entry point property on `neutrino.options.mains` now returns a normalized object, even when using string options.
+- **BREAKING CHANGE** When configuring webpack with a target of `node` the `@neutrinojs/jest` preset will default its
+`testEnvironment` to also be `node` instead of `jsdom` [#1030](https://github.com/neutrinojs/neutrino/pull/1030).
+- **BREAKING CHANGE** Various dependencies have been updated which may bring their own breaking changes. Please
+check and test your project to ensure proper functionality.
+- ESLint caching is now enabled by default, so it is recommended to specify `.eslintcache` as being ignored from
+your source control commits.
+
```

Comment:
Should we add a link to the full list of changes? (eg [https://github.com/neutrinojs/neutrino/compare/v8.3.0...master](https://github.com/neutrinojs/neutrino/compare/v8.3.0...master) but updated to use the v9.0.0 tag when it exists)

Reply:
Sure.

Reply:
This is just `[]` for now; should we change this to use `...master` for now, then update later?

Reply:
Yeah, that works for me.